src/Lime.Messaging/Contents/Invoice.cs

```diff
@@ -92,5 +104,14 @@ public Invoice()
         /// </value>
         [DataMember(Name = ITEMS_KEY)]
         public InvoiceItem[] Items { get; set; }
+
+        /// <summary>
+        /// Get or sets the invoice customer to be billed.
+        /// </summary>
+        /// <value>
+        /// The contact billed to
+        /// </value>
+        [DataMember(Name = BILLED_TO_KEY)]
+        public Contact BilledTo { get; set; }
```

Comment:
Use `billTo` instead.

```suggestion
        public Contact BillTo { get; set; }
```